packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -37,6 +37,29 @@ describe('MultiSelect', () => {
       await expect(container).toHaveNoACViolations('MultiSelect');
     });
   });
+  it('does not render items with undefined values', async () => {
+    const items = [{ text: 'joey' }, { text: 'johnny' }, { text: undefined }];
+    const label = 'test-label';
+    const { container } = render(
+      <MultiSelect
+        id="custom-id"
+        label={label}
+        items={items}
+        itemToString={(item) => (item ? item.text : '')}
+      />
+    );
+    // eslint-disable-next-line testing-library/prefer-screen-queries
+    const labelNode = getByText(container, label);
```

Comment:
This should be possible with the allowed 
```js
screen.getByRole('combobox', { name: label })
```

Reply:
shortened this to just screen.getByRole('combobox') becuase name is coming in as  empty string and giving error as  :
TestingLibraryElementError: Unable to find an accessible element with the role "combobox" and name "test-label"

    Here are the accessible roles:

      combobox:

      Name "":
      <button
        aria-activedescendant=""
        aria-controls="downshift-:r0:-menu"
        aria-expanded="false"
        aria-haspopup="listbox"
        aria-labelledby="downshift-:r0:-label"
        class="cds--list-box__field"
        id="downshift-:r0:-toggle-button"
        role="combobox"
        tabindex="0"
        type="button"
      />